.gitignore

```diff
@@ -1,5 +1,26 @@
+<<<<<<< HEAD
```

Comment:
Will fix, this is basically a completely new code base that was rebased on `master`.